src/SprykerEco/Yves/FactFinderWebComponents/Model/WebComponentsConfigProvider/WebComponentsConfigProviderInterface.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\FactFinderWebComponents\Model\WebComponentsConfigProvider;
+
+interface WebComponentsConfigProviderInterface
```

Comment:
Why do we need this?

Reply:
I've removed a class which implemented this interface. Now FactFinderWebComponentsConfig returns this options.